lib/networking/kustomization.yaml

```diff
@@ -0,0 +1,367 @@
+      name: node-2
+    fieldPaths:
+    - spec.desiredState.interfaces.[name=tenant].ipv4.address.0.ip
+- source:
+    kind: ConfigMap
+    name: network-values
+    fieldPath: data.node_2.ctlplane_ip
+  targets:
+  - select:
+      kind: NodeNetworkConfigurationPolicy
+      name: node-2
+    fieldPaths:
+    - spec.desiredState.interfaces.[type=ethernet].ipv4.address.0.ip
+- source:
+    kind: ConfigMap
+    name: network-values
+    fieldPath: data.node_2.storage_ip
+  targets:
+  - select:
+      kind: NodeNetworkConfigurationPolicy
+      name: node-2
+    fieldPaths:
+    - spec.desiredState.interfaces.[name=storage].ipv4.address.0.ip
+
+# Node names
+- source:
+    kind: ConfigMap
+    name: network-values
+    fieldPath: data.node_0.name
+  targets:
+  - select:
+      kind: NodeNetworkConfigurationPolicy
+      name: node-0
+    fieldPaths:
+    - metadata.name
+    - spec.nodeSelector.[kubernetes.io/hostname]
+- source:
+    kind: ConfigMap
+    name: network-values
+    fieldPath: data.node_1.name
+  targets:
+  - select:
+      kind: NodeNetworkConfigurationPolicy
+      name: node-1
+    fieldPaths:
+    - metadata.name
+    - spec.nodeSelector.[kubernetes.io/hostname]
+- source:
+    kind: ConfigMap
+    name: network-values
+    fieldPath: data.node_2.name
+  targets:
+  - select:
+      kind: NodeNetworkConfigurationPolicy
+      name: node-2
+    fieldPaths:
+    - metadata.name
+    - spec.nodeSelector.[kubernetes.io/hostname]
+
+# Loadbalancer address pools
+- source:
+    kind: ConfigMap
+    name: network-values
+    fieldPath: data.ctlplane.lb_addresses
+  targets:
+  - select:
+      group: metallb.io
+      kind: IPAddressPool
+      name: ctlplane
+    fieldPaths:
+    - spec.addresses
+- source:
+    kind: ConfigMap
+    name: network-values
+    fieldPath: data.internalapi.lb_addresses
+  targets:
+  - select:
+      group: metallb.io
+      kind: IPAddressPool
+      name: internalapi
+    fieldPaths:
+    - spec.addresses
+- source:
+    kind: ConfigMap
+    name: network-values
+    fieldPath: data.tenant.lb_addresses
+  targets:
+  - select:
+      group: metallb.io
+      kind: IPAddressPool
+      name: tenant
+    fieldPaths:
+    - spec.addresses
+- source:
+    kind: ConfigMap
+    name: network-values
+    fieldPath: data.ctlplane.lb_addresses
+  targets:
+  - select:
+      group: metallb.io
+      kind: IPAddressPool
+      name: ctlplane
+    fieldPaths:
+    - spec.addresses
+- source:
+    kind: ConfigMap
+    name: network-values
+    fieldPath: data.storage.lb_addresses
+  targets:
+  - select:
+      group: metallb.io
+      kind: IPAddressPool
+      name: storage
+    fieldPaths:
+    - spec.addresses
+
+# Loadbalancer interfaces
+- source:
+    kind: ConfigMap
+    name: network-values
+    fieldPath: data.ctlplane.iface
+  targets:
+  - select:
+      group: metallb.io
+      kind: L2Advertisement
+      name: ctlplane
+    fieldPaths:
+    - spec.interfaces.0
```

Comment:
I've opened a PR against your branch to add other interfaces: https://github.com/fultonj/architecture/pull/1/files